README.md

```diff
@@ -1,46 +1,121 @@
 # opencontainers.org
 
-This repository contains a boilerplate static site generator setup for creating CNCF documentation projects. We strongly recommend using this setup (it helps us help you and your project!), but none of the technologies in the stack are strictly required.
+This repository contains the markdown and static site generator for the https://opencontainers.org website. 
 
 The starter uses the following:
 * **[Hugo](https://gohugo.io/)** as a static site generator
-* **[Bulma](https://bulma.io/)** as a CSS framework
-* **[Netlify](https://www.netlify.com/)** for building, hosting, and DNS management
+* **[Tailwind CSS](https://tailwindcss.com/)** as a CSS framework
+* **[Netlify](https://www.netlify.com/)** for building and hosting.
+
+# Contributing 
+
+To make changes to https://opencontainers.org, create a fork of this repository and submit a pull request. Doing so successfully generates a preview URL using Netlify. 
+
+## Prerequisites 
+
+- Read and abide by the [code of conduct](https://github.com/opencontainers/.github/blob/master/CODE_OF_CONDUCT.md)
+- Sign the [CNCF CLA](https://github.com/kubernetes/community/blob/master/CLA.md)
 
 ## Running locally
 
-Make sure you have [npm](https://www.npmjs.com/) and [yarn](https://yarnpkg.com/) installed. Clone this repository and run the following two commands in its directory:
+Install [npm](https://www.npmjs.com/) and [yarn](https://yarnpkg.com/), then run the following commands in your local repository:
 
 ```shell
-# Install npm assets (just Bulma for Sass/CSS)
 yarn
-
 # Run the server locally
 make serve
 ```
+This installs Hugo and all other dependencies.
+
+## Writing blog posts, news updates, and announcements
+
+The OCI has 3 categories of posts, all stored in the `/content/posts` directory:
+
+- `/news`: news stories from external sources about the OCI
+- `/blog`: blog posts published about by the OCI
+- `/announcements`: press releases published by the OCI or Linux Foundation for distribution
+
+When writing a post:
+
+1. Pick the appropriate category for your post 
+2. Create a markdown file. To keep things easy for others, name it in the following format: `YYYY-MM-DD-your-title-here.md`. 
+3. Use the following frontmatter tags (examples below):
+
+  Tag | Description | Notes |
```

Comment:
Markdown table nit: remove trailing pipes, both for readability and to prevent a parser from mistaking the possibility of an additional column. Apply to all lines in the table.
```suggestion
  Tag | Description | Notes
```